### DISCUSSION TITLE
THIS PROJECT IS UNMAINTANED AGAIN. (Over 400+ neglected pull request)

### DISCUSSION THREAD

**Discussion by w3teal** [abhisheknaiidu/awesome-github-profile-readme#1199] [General]
(maybe)

This is was an cool project, and i was intend to add my README profile here. But I just realized that a lot unmerged pull request. And i saw that the last pull request is in 2 years ago (#907), and i never see the another after it. And also in the [Product Hunt page](https://www.producthunt.com/products/awesome-github-profiles?utm_source=badge-featured&utm_medium=badge#awesome-github-profiles) (hover on the ghost in the name) said "Unfortunately this product is no longer available.". This is so sad. But the creator of this project (@abhisheknaiidu) still active to make contribution on another repository. Meanwhile the owner of the website (@zzetao) not active since last year.

**Comment by ccoVeille** [comment]
Hi @LIGMATV 

I agree with you this repository is unfortunately not maintained.

I like your initiative

For records @nikhil-shuklaa also shared the same idea.

https://dev.to/nikhilshukla/showcase-your-github-profile-contribute-to-the-cool-github-profile-readme-repository-4c1p

https://github.com/nikhil-shuklaa/Cool-GitHub-Profile-README

I'm bringing your attention on the fact your inspirations might be close.

Maybe you could collaborate.

Christophe

**Reply by ccoVeille** [reply]
@roypriyanshu02 might be also interested as you quoted his repository

https://github.com/roypriyanshu02/impressive-profile-readmes

**Reply by nikhilsbuilds** [reply]
Hey @ccoVeille, Thanks :) 
I'm working on the website and will deploy it soon, but it is open for contributions.

**Reply by roypriyanshu02** [reply]
Hey @ccoVeille and @LIGMATV, thanks!

I'm excited to share that I have plans to keep this project alive and continually improve it. Although I'm currently busy with personal commitments, I have a roadmap to enhance performance (data regeneration and deployment speed), bring on more maintainers, improve the UI/UX, and SEO, and add more features and improvements!

**Reply by roypriyanshu02** [reply]
As part of the upcoming improvements, I'm looking to bring on additional maintainers to help keep the project running smoothly. Would any of you be interested in becoming a maintainer? Your contributions would be greatly appreciated!

**Reply by ccoVeille** [reply]
I'm very happy anyway that you came to the same conclusion than me. I didn't share it until now, but maybe it was obvious to you by reading me: I think this kind of project can only maintained by multiple maintainers on a long term period.

That's why I wanted to bring you altogether, and create a discussion.

I know that's often a problem for people to jump into someone else project, because the project will stay owned only by the initial owner. I would like you to suggest to consider creating a GitHub organization, and move one of the project (the one with more stars usually) to the organization where all people will become maintainer, with a dedicated "owner"

**Reply by ccoVeille** [reply]
> Would any of you be interested in becoming a maintainer?

I would decline anyway, because I'm busy also. And I would like to focus on Go tool/library and also a personal project I try to launch ( https://github.com/ccoveille/jargon )

**Reply by roypriyanshu02** [reply]
@ccoVeille no problem! Best wishes for your projects.

**Reply by roypriyanshu02** [reply]
> "I think this kind of project can only be maintained by multiple maintainers on a long-term period."

I agree—it’s good for long-term sustainability, faster and better quality decision-making, and helps to create a centralized source for others.

> "I know that's often a problem for people to jump into someone else's project, because the project will stay owned only by the initial owner. I would like you to suggest to consider creating a GitHub organization"

If anyone wants to do it, I'm totally in for it. @nikhil-shuklaa, @LIGMATV, or anyone else, let me know your thoughts.
